src/components/Shop/query.ts

```diff
@@ -35,3 +38,22 @@ const shopInfo = gql`
   }
 `;
 export const TypedShopInfoQuery = TypedQuery<ShopInfo, {}>(shopInfo);
+
+const limitInfo = gql`
+  ${limitFragment}
+  query RefreshLimits(
+    $channels: Boolean!
+    $orders: Boolean!
+    $productVariants: Boolean!
+    $staffUsers: Boolean!
+    $warehouses: Boolean!
```

Comment:
Use a default value of `false` for each of those variables, this will make the query itself more readable (without having to manually set everything except one to `false`)

Reply:
Oooooooh